triage-actions/api/api.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+Object.defineProperty(exports, "__esModule", { value: true });
+//# sourceMappingURL=api.js.map
```

Comment:
This seems weird.  Did you comment this out or was it copied that way?

Reply:
Yeah the .js files are automatically generated - any time you run a build and I think as a pre-commit hook. You would see this comment on the .js files in our extension, too - we just don't commit them to GitHub normally. In this case, we commit them because the bot runs directly from the repo, as opposed to our extension which gets packaged into a vsix